feat(shows): roughs in shows2

dzucconi opened the pull request:
### TODO

- [ ] Index shows infinite scroll / artwork masonry
- [ ] City shows pagination
- [ ] Online "city" page
- [ ] Alternate date display for ending today

![localhost_5000_shows2](https://user-images.githubusercontent.com/112297/124830486-f326a400-df47-11eb-876f-d70a032a2d0e.png)

dzucconi commented on src/v2/Apps/Shows/Components/ShowsHeader.tsx:
```diff
@@ -0,0 +1,139 @@
+
+const DEFAULT_CITIES = [
+  { text: "All", value: "" },
+  // { text: "Online Exclusive", value: "online" }, // TODO
```
@anandaroop how bad of an idea do you think it is to resolve a fictional `online` city in the Metaphysics `CityType`

![keyboard_surfing_the_internet2-406x450](https://user-images.githubusercontent.com/112297/106330878-05dad580-6252-11eb-8808-9ace5be932f6.jpg)

anandaroop replied:
Whoa, `cyberspace`… 

It feels awkward to me, and I'm wondering if there's a legit user need for it. But what would happen here — a listing page for all online-exclusive shows? Maybe that could be better handled with its own landing page?

dzucconi replied:
We effectively treat "online" as a city in the frontend: https://www.artsy.net/shows/online

In general I think it makes sense to push any awkwardness into the graph; but it's weird in this case; admittedly. The alternative is to build out the args for the top-level shows connection in Metaphysics then create a copy of the city page with different queries.

anandaroop replied:
Ah, I see. It will be odd that `online` resolves to a City that has no coordinates, but weirdly the schema seems to allow that. 

Just to confirm this will purely be a matter of Force and MP agreeing on this `online` convention, no support needed in Gravity, yeah?

If so I can get with this (barring a re-think to how we handle online shows on the front end to begin with).

dzucconi replied:
Yeah; just resolving a little object with a slug/name for online then hardcoding an exception for it's showsConnection. 

But, I'm going to PR some additions to the top-level showsConnection anyway, because might as well see how awkward it really is (maybe it's not so bad 🤞 ).

oxaudo replied:
Hah... This 'online' hack just keeps on giving!

joeyAghion replied:
It's weird to have an `online` "city" but less weird to call the same value "location" (if we could accomplish that across both Force and MP logic).

ArtsyOpenSource commented:
<!--
  0 failure: 
  1 warning:  Routes added to `...
  
  
  DangerID: danger-id-Danger;
-->


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 11ec394a41399eaa33b69fa3ace344bef549c0be
</p>

dzucconi commented:
Rebased and updated to V3 (though I'll probably need some additional guidance). This just adds a new /shows2 route and so is fine to merge once green. Will follow up with those TODOs.
